mantidimaging/core/operations/rescale/rescale.py

```diff
@@ -44,9 +47,17 @@ def filter_func(images: Images,
         return images
 
     @staticmethod
-    def filter_single_image(image: ndarray, min_input: float, max_input: float, max_output: float, data_type=float32):
+    def filter_single_image(image: ndarray,
+                            min_input: float,
+                            max_input: float,
+                            max_output: float,
+                            offset: float,
+                            data_type=float32):
+        if offset < 0:
+            raise ValueError("The offset value provided must not be negative.")
         image[image < min_input] = 0
         image[image > max_input] = 0
```

Comment:
Clearing the out of range values will need to happen after rescaling now, because for some cases zero will be a value in range.

e.g. If the data spans from -10 to +10. and I set the limits to -5 to +5. then a pixel at -8 will get set to zero, then scaled into the middle of the final output range